src/viewer/Viewer.js

```diff
@@ -82,7 +82,10 @@ class Viewer extends Component {
   componentDidMount() {
     document.body.addEventListener(EVENT_KEYDOWN, this.onKeyDown);
 
-    this.getNextCase();
+    // there is no stand alone viewer. Flow comes from dashboard to viewer.
+    if (this.props.collection) {
```

Comment:
When would this occur? The viewer shouldn't be able to be opened without a collection, unless maybe the user has finished the collection. What do you mean by standalone viewer?